src/views/Editor.vue

```diff
@@ -319,6 +320,10 @@ export default {
 			await this.$auth.logoutDueSessionTimeout()
 			this.$router.push({ name: "home", params: { missingSession: true }})
 		},
+		async handleOtherError() {
+			this.otherError=true
+			this.$router.push({ name:"datasets", params: { otherError: true }})
```

Comment:
why this?

Reply:
redirecting to dataset page with otherError flag

Reply:
why do we redirect?

Reply:
Error is thrown from editor page..
As per story, we are not logging out the user. and we can not show editor page as it is errored. so best place would be dataset page

Reply:
what happens when the user is on a train and there is a bad/poor connection and when the user has plenty of datasets.